x-pack/plugins/cloud_security_posture/public/pages/rules/rules_flyout.tsx

```diff
@@ -74,6 +87,12 @@ export const RuleFlyout = ({ onClose, rule, refetchRulesStates }: RuleFlyoutProp
       const nextRuleStates = isRuleMuted ? 'unmute' : 'mute';
       await postRequestChangeRulesStates(nextRuleStates, [rulesObjectRequest]);
       await refetchRulesStates();
+      await showChangeBenchmarkRuleStatesSuccessToast(
+        notifications,
+        nextRuleStates !== 'mute',
```

Comment:
this `nextRuleStates` is quite confusing. I guess it shouldn't be plural, plus state for me is smth like `muted | unmuted` while `mute | unmute` is action, not a state. But in the end , why do you need it here anyway? you already have `isRuleMuted`, why don't use just use it? Now you are getting the next state from the current, but then getting the current from the next.

Reply:
woops you are right